README.md

```diff
@@ -5,11 +5,15 @@ Project structure based on https://github.com/golang-standards/project-layout
 # The demo
 ## Prerequisites
 * [Docker](https://www.docker.com/community-edition#/download)
+* [Go](https://golang.org/doc/install) and [goimports](https://godoc.org/golang.org/x/tools/cmd/goimports)
 
 ## Running the demo
 ```bash
-git@github.com:ewilde/kubecon.git ewilde-kubecon
-cd ewilde-kubecon
+export GOPATH=~/go  # or your alternative Go-language path
+REPOPATH="$GOPATH"/src/github.com/ewilde/kubecon
+mkdir -p "$REPOPATH"
+git clone https://github.com/ewilde/kubecon.git "$REPOPATH"
+cd "$REPOPATH"
```

Comment:
Maybe my Go installation/setup is misconfigured (missing env vars?), but I found that if the git repo was cloned outside of `~/go/src/`, then the `go` commands run by `make build` would fail. In particular, `go list ./...` would return something like `_/Users/paulo/ewilde-kubecon` (note the underscore prefix) instead of `github.com/ewilde/kubecon/cmd/http-echo`, and this would cause the `errcheck` command to fail.

Reply:
@paulo-castro yes I assume you check out the repo inside your go path, nice update to the README.md